packages/vega-transforms/src/Window.js

```diff
@@ -37,6 +38,7 @@ Window.Definition = {
     { 'name': 'groupby', 'type': 'field', 'array': true },
     { 'name': 'ops', 'type': 'enum', 'array': true, 'values': ValidWindowOps.concat(ValidAggregateOps) },
     { 'name': 'params', 'type': 'number', 'null': true, 'array': true },
+    { 'name': 'aggregate_params', 'type': 'field', 'null': true, 'array': true, 'default': [null] },
```

Comment:
Is the default correct here? There may be more than two ops.

Reply:
The code will pad aggregate_params with nulls as necessary to match the number of ops.